### PR TITLE
Add test case for fine-tuning the BERT model with link prediction on graphs with featureless nodes.

### PR DESCRIPTION
**Pull request by classicsong**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/source/configuration/configuration-run.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ Model Configurations
 --------------------------------
 GraphStorm provides a set of parameters to config the GNN model structure (input layer, gnn layer, decoder layer, etc)
 
-- **model_encoder_type**: (**Required**) Graph encoder model used to encode graph data. It can be rgat or rgcn.
+- **model_encoder_type**: (**Required**) The Encoder model used to encode graph data. It can be a GNN encoder or a non-GNN encoder. A GNN encoder is composed of an input layer, which encodes input node features, and a GNN layer. A non-GNN encoder only contains an input layer. GraphStorm supports two GNN encoders: `rgcn` which uses relational graph convolutional network as its GNN model and `rgat` which uses relational graph attention network as its GNN model. GraphStorm supports two non-GNN encoder: `lm` which requires each node type has and only has text features and uses language model, e.g., Bert, to encode these features and `mlp` which accepts various types of input node features (text feature, floating points and learnable embeddings) and finally uses an MLP to project these features into same dimension.
```
Comment (zheng-da) [inline docs/source/configuration/configuration-run.rst:46]: let's use "input module" and "GNN module". the term might be confusing.

File: training_scripts/gsgnn_lp/ml_lm_input_lp.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      node_types:
+        - movie
+gsf:
+  basic:
+    backend: gloo
+    ip_config: ip_list.txt
+    part_config: movielen_100k_lp_user_feat_movie_token_1p/ml.json
+    verbose: false
+    save_perf_results_path: null
+  gnn:
```
Comment (zheng-da) [inline training_scripts/gsgnn_lp/ml_lm_input_lp.yaml:18]: this is confusing. it says gnn, but the encoder is mlp.

File: docs/source/configuration/configuration-run.rst (hunk 1 above)
Comment (zheng-da) [reply docs/source/configuration/configuration-run.rst:46]: we also need to discuss what is the intuitive way to specify these. i think the current arguments are confusing. i don't know how to specify them either.

File: training_scripts/gsgnn_lp/ml_lm_input_lp.yaml (hunk 2 above)
Comment (classicsong) [reply training_scripts/gsgnn_lp/ml_lm_input_lp.yaml:18]: changed

**Review by zheng-da — APPROVED** [review]

**Pull request merged**